documentation/specs/custom-cultures.md

```diff
@@ -0,0 +1,36 @@
+# MSBuild Custom Cultures Support
+
+## Overview
+
+The `MSBUILDENABLECUSTOMCULTURES` feature flag provides an opt-in mechanism for handling custom culture-specific resources in MSBuild projects. This feature allows for greater control over which directories are treated as culture-specific resources during the build process.
+
+## Purpose
+
+In some projects, directory names that match culture name patterns might not actually be culture resources. This can cause issues with resource compilation and deployment. This feature flag enables:
+
+1. Control over whether custom culture detection is enabled
+2. Fine-grained configuration of which directories should be excluded from culture-specific resource processing
+
+## Usage
+
+### Enabling the Feature
+
+To enable the custom cultures feature, set the `MSBUILDENABLECUSTOMCULTURES` environment variable to `1` in your project file:
```

Comment:
Did you mean set an environment variable without the "in your project file"?

and a dangling colon